Fix building of 3rd party applications. There is a "struct cdrom_drive" used
in cdda_paranoia.h but it doesn't not do a dummy struct define (or include
cdda_interface.h for it as both of these headers are installed). Thanks to
Mike Frysinger <vapier AT gentoo.org> for explanation. See,

http://bugs.gentoo.org/257607 (downstream bug)

NOTE for upstream: This also needs -I../interface at build time in CFLAGS
which we do inside ebuild but if this is applied it should come from the
build system.

diff -ur cdparanoia-III-10.2.orig/paranoia/cdda_paranoia.h cdparanoia-III-10.2/paranoia/cdda_paranoia.h
--- cdparanoia-III-10.2.orig/paranoia/cdda_paranoia.h	2008-09-11 23:09:40.000000000 +0300
+++ cdparanoia-III-10.2/paranoia/cdda_paranoia.h	2009-06-20 21:12:44.000000000 +0300
@@ -39,6 +39,7 @@
 #endif
 
 #include <stdio.h>
+#include <cdda_interface.h>
 
 extern char *paranoia_version();
 extern cdrom_paranoia *paranoia_init(cdrom_drive *d);
